<launch>
  <!-- Launch the rfid reader -->
  <node pkg="rfid_node" type="rfid_reader_ros.py" name="rfid_node" output="screen" respawn="true">
      <rosparam param="RFIDdeviceURI">tmr:///dev/rfid</rosparam>      
 </node>
 
 </launch>
